### PR TITLE
Assign MS as audio conferencing provider

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/4494

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/set-up-audio-conferencing-in-teams.md [hunk 1]
```diff
@@ -48,6 +48,9 @@ Go to [Country and region availability for Audio Conferencing and Calling Plans]
     > Audio Conferencing is included in Office 365 Enterprise E5 licenses and as an add-on.
         
 2. After you buy the Audio Conferencing licenses, you will need to assign them to those people in your organization who are going to schedule or lead meetings. See [Assign licenses to users in Office 365 for business](https://support.office.com/article/997596b5-4173-4627-b915-36abac6786dc) you purchased to the people in your organization who are going to schedule or lead meetings.
+
+> [!IMPORTANT]
+> Make sure you assign Microsoft as the audio conferencing provider. More details [here](https://docs.microsoft.com/en-us/SkypeForBusiness/audio-conferencing-in-office-365/assign-microsoft-as-the-audio-conferencing-provider).
```
Comment (JohanFreelancer9) [inline Teams/set-up-audio-conferencing-in-teams.md:53]:
```suggestion
> Make sure you [assign Microsoft as the audio conferencing provider](https://docs.microsoft.com/SkypeForBusiness/audio-conferencing-in-office-365/assign-microsoft-as-the-audio-conferencing-provider).
```

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by tonysmit** [comment]
@scanum and @JohanFreelancer9  - please don't accept this change. Thanks, Tony

**Pull request closed without merging**